src/vivarium_census_prl_synth_pop/components/population.py

```diff
@@ -192,14 +194,25 @@ def choose_standard_households(self, target_number_sims: int) -> pd.DataFrame:
             how="left",
         )
 
-        # get rid simulants in excess of desired pop size
+        # get rid of simulants and households in excess of desired pop size
         households_to_discard = chosen_persons.loc[
-            target_number_sims:, "household_id"
+            target_number_sims:, "census_sample_household_id"
         ].unique()
-
         chosen_persons = chosen_persons.loc[
-            ~chosen_persons["household_id"].isin(households_to_discard)
+            ~chosen_persons["census_sample_household_id"].isin(households_to_discard)
```

Comment:
`households_to_discard` might actually be bigger than `households_to_keep`. If so, it would be more performant to do 
```
chosen_persons.loc[
    chosen_persons["census_sample_household_id"].isin(households_to_keep)
]
```
and same for households below.

Reply:
IIRC, the mean household size was around 2, so I think they are about the same size. I'm going to keep this how it is because it seems easier to find `households_to_discard` than `households_to_keep`.